arches/app/src/components/ControlledListManager/ListHeader.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import type { ControlledList } from "@/types/ControlledListManager.d";
+
+const { $gettext } = useGettext();
+const slateBlue = "#2d3c4b"; // todo: import from theme somewhere
+
+const props: {
+    displayedList: ControlledList;
+    isItemEditor: boolean;
+} = defineProps([
+    "displayedList",
+    "isItemEditor",
+]);
+
+const heading = computed(() => {
+    const prefix = props.isItemEditor ? $gettext("List Item Editor") : $gettext("List Editor");
+    return (
+        prefix + (props.displayedList ? " > " + props.displayedList.name : "")
```

Comment:
This may not work for all languages and should be changed. Also if possible avoid empty ternary expressions